sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/transform/BeamBuiltinAggregations.java

```diff
@@ -171,10 +173,18 @@ static CombineFn createAvg(Schema.FieldType fieldType) {
         return new BigDecimalAvg();
       default:
         throw new UnsupportedOperationException(
-            String.format("[%s] is not support in AVG", fieldType));
+            String.format("[%s] is not supported in AVG", fieldType));
     }
   }
 
+  static CombineFn createBitOr(Schema.FieldType fieldType) {
+    if (fieldType.getTypeName() == TypeName.INT64) {
+      return new BitOr();
+    }
+    throw new UnsupportedOperationException(
+        String.format("[%s] is not supported in BIT_OR", fieldType));
```

Comment:
Ah I didn't realized that Calcite does not do type validation on argument type for a specific function. ZetaSQL does it.

Reply:
I saw the other functions in this class do something similar, so I thought I'd do so as well